examples/terms-modal-banner-dynamodb/README.md

```diff
@@ -66,14 +66,14 @@ Once you have finished all of the steps related to AWS in the [developer tutoria
 
 1. In your new Miro app settings, add in the `App URL`. It should end with `webassets/app.html`.
 2. Open the [app manifest editor](https://developers.miro.com/docs/manually-create-an-app#step-2-configure-your-app-in-miro) by clicking **Edit in Manifest**. \
-   In the app manifest editor, configure the app as follows <b> ⚠️ Note: you will need to change your sdkUri to be your app.html which is hosted in the `webassets` folder of your S3 bucket. Mine is just shown as an example </b> and then click save:
+   In the app manifest editor, configure the app as follows <b> ⚠️ Note: you will need to change your sdkUri to be your app.html which is hosted in the `webassets` folder of your S3 bucket. Your `sdkUri` will be the public endpoint to your app.html file. Mine is just shown as an example </b> so make sure to change it to point to your `app.html file` and then click save:
```

Comment:
_personal opinion warning_
Im not sure about using the first person format in the readme's. Maybe we can better change to the 'We' form?

Reply:
I'll leave this be for now, but will change it if needed in another PR. don't think the effort is worth it in this case :)